src/v/kafka/server/usage_manager.h

```diff
@@ -82,7 +83,7 @@ class usage_manager : public ss::peering_sharded_service<usage_manager> {
 
     /// Obtain all current stats - for all shards
     ///
-    std::vector<usage_window> get_usage_stats() const;
+    ss::future<std::vector<usage_window>> get_usage_stats(bool include_open);
```

Comment:
Looks like `include_open` is always equal to true up until this point in the PR. Do we really need this boolean switch?

Reply:
the value comes via the initial HTTP request to `/v1/usage`

Reply:
Right, but the downstream system will always call it with `include_open` right?

Reply:
@alenkacz is there a need to have `include_open` as a parameter?

Reply:
are you asking whether we need HTTP parameter `include_open_bucket` or whether we include open bucket on every request and drop that parameter?
I thought that param is needed also from redpanda point of view because we do an extra read on the storage when called with this param and we don't want to do it for every request.

Reply:
Yeah on redpandas side its fine to just drop this altogether and include open bucket in all responses, simplifies things.